core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -56,6 +56,8 @@ oppia.directive('editorNavigation', [
           });
 
           $scope.userIsLoggedIn = GLOBALS.userIsLoggedIn;
+          // This will be removed after translation tab will be ready.
+          $scope.enableTranslationTab = true;
```

Comment:
Set this to false. Actually, I think it may be better to put this in an oppia.constant? Just so it's more visible and we don't forget about it.

Reply:
Done!